### ISSUE TITLE
nsm clone fail

### ISSUE THREAD

**Issue by Scicomath** [nsm-templates/tiar-site#1]
When I try nsm clone this repo, I get following error:
~ nsm clone https://github.com/nsm-templates/tiar-site.git
Cloning into 'tiar-site'...
remote: Enumerating objects: 81, done.
remote: Counting objects: 100% (81/81), done.
remote: Compressing objects: 100% (51/51), done.
remote: Total 81 (delta 31), reused 79 (delta 29), pack-reused 0
Unpacking objects: 100% (81/81), done.
opening project config file: '.nsm/nift.config'
opening project tracking file: '.nsm/tracking.list'
🔥: clone: system('git checkout -- .') failed in '/Users/aixin/.temp-output-dir'

**Comment by n-ham** [comment]
This seems to be an issue with cloning quite a few (though not all) template repos. I am looking in to this right away..

**Comment by n-ham** [comment]
Though so, the problem is with the new way I am copying directories that isn't just using a system call to copy them. Will try to work out what files aren't copying properly and fix it asap!

**Comment by n-ham** [comment]
[Here's](https://github.com/nifty-site-manager/nsm/blob/d84332973a9c5ff67998c02648ce3be724be9740/FileSystem.cpp#L437) the problem:

```
if(targetDir[sourceDir.size()-1] == '/' || targetDir[targetDir.size()-1] == '\\')
```

should be

```
if(targetDir[targetDir.size()-1] == '/' || targetDir[targetDir.size()-1] == '\\')
```

I will release a fixed version in the next few hours (might do a couple of other things first).

**Comment by n-ham** [comment]
I just released v2.3.4 which should have this fixed.
